web3.js/src/layout.ts

```diff
@@ -4,24 +4,47 @@ import * as BufferLayout from '@solana/buffer-layout';
 /**
  * Layout for a public key
  */
-export const publicKey = (
-  property: string = 'publicKey',
-): BufferLayout.Layout => {
+export const publicKey = (property: string = 'publicKey') => {
   return BufferLayout.blob(32, property);
 };
 
 /**
  * Layout for a 64bit unsigned value
  */
-export const uint64 = (property: string = 'uint64'): BufferLayout.Layout => {
+export const uint64 = (property: string = 'uint64') => {
   return BufferLayout.blob(8, property);
 };
```

Comment:
My initial inclination was that we could maybe use + reexport from `@solana/buffer-layout-utils` for these, but I guess that's breaking, so scratch that.